test-infrastructure/jira-container/README.md

```diff
@@ -0,0 +1,50 @@
+# Jira SDK Container
+
+This code is intended to be use with testing the alerter code, in the future it could be augmented to take command line arguments to load different configs scripts to seed the Jira state for each application we are trying to test.
+
+The Jira instance will have a license to allow you to test for 3 days from starting the container. After 3 days you will have to launch a new instance with a fresh state. 
+
+Currently we install the jira software app and mirror much of prod Jira, however workflows can not be created via the Jira API and macfc-workflow.xml has been included in this directory and can be optionally imported using the jira web UI.
+
+## Build and Run
+
+`docker build -t jira-atlassian-sdk .`
```

Comment:
To get this to build successfully on my Mac I had to add `--platform linux/amd64`. Is it worth specifying this in the README?

Reply:
Updated